From ff477d2b3f5ddc582140e6d357e52f1cb1da74ce Mon Sep 17 00:00:00 2001
Message-Id: <ff477d2b3f5ddc582140e6d357e52f1cb1da74ce.1555699269.git.jan.steffens@gmail.com>
From: Jon Turney <jon.turney@dronecode.org.uk>
Date: Sat, 30 Mar 2019 18:35:00 +0000
Subject: [PATCH] Fix 'failing/85 gtest dependency with version' test to fail
 on Arch

If gtest is patched to have a pkg-config file, that will report the
version, so force the 'system' method to be used when we are exercising
that an unknown version doesn't satisfy any version constraint.
---
 test cases/failing/85 gtest dependency with version/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test cases/failing/85 gtest dependency with version/meson.build b/test cases/failing/85 gtest dependency with version/meson.build
index 5115f279..3d909942 100644
--- a/test cases/failing/85 gtest dependency with version/meson.build	
+++ b/test cases/failing/85 gtest dependency with version/meson.build	
@@ -1,3 +1,3 @@
 project('gtest dependency with version', ['c', 'cpp'])
 # discovering gtest version is not yet implemented
-dep = dependency('gtest', version: '>0')
+dep = dependency('gtest', method: 'system', version: '>0')
-- 
2.21.0

